docs/index.stories.tsx

```diff
@@ -468,3 +468,23 @@ export const BeforeMask = () => {
     </>
   )
 }
+
+/**
+ * Animation begin delay
+ */
+export const animateBegin = () => {
+  return (
+    <>
+      <SyntaxCode>{' <ContentLoader animateBegin="5s" />'}</SyntaxCode>
+      <ContentLoader animateBegin="5s" />
+    </>
+  )
+}
+
+animateBegin.story = {
+  parameters: {
+    notes: `##\`animateBegin?: string\` 
+    
+    Defaults to \`0s\`. This attribute accepts the same values as the SVG animate element's [\`begin\` attribute](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/begin).`,
```

Comment:
Note default to `0s` is only implicit here; the actual default value lives with the `begin` attribute itself on the `animation` element.